lib/src/Base/Optim/Dlib.cxx

```diff
@@ -645,28 +640,30 @@ void Dlib::run()
                          optimPoint,
                          SpecFunc::LowestScalar );
     }
-    else
-    {
-      if (getProblem().hasBounds())
-        dlib::find_max_box_constrained (searchStrategy,
-                                        stopStrategy,
-                                        objectiveDlibFunction,
-                                        objectiveDlibGradient,
-                                        optimPoint,
-                                        lb,
-                                        ub);
-      else
-        dlib::find_max (searchStrategy,
-                        stopStrategy,
-                        objectiveDlibFunction,
-                        objectiveDlibGradient,
-                        optimPoint,
-                        SpecFunc::MaxScalar);
-    }
+//     else
+//     {
+//       if (getProblem().hasBounds())
+//         dlib::find_max_box_constrained (searchStrategy,
+//                                         stopStrategy,
+//                                         objectiveDlibFunction,
+//                                         objectiveDlibGradient,
+//                                         optimPoint,
+//                                         lb,
+//                                         ub);
+//       else
+//         dlib::find_max (searchStrategy,
+//                         stopStrategy,
+//                         objectiveDlibFunction,
+//                         objectiveDlibGradient,
+//                         optimPoint,
+//                         SpecFunc::MaxScalar);
+//     }
+
```

Comment:
Dead code?

Reply:
yes, replace that with an explanation of why we dont use find_max